codex/market.nim

```diff
@@ -84,6 +84,10 @@ method getRequestEnd*(market: Market,
                       id: RequestId): Future[SecondsSince1970] {.base, async.} =
   raiseAssert("not implemented")
 
+method getRequestExpiry*(market: Market,
```

Comment:
I'm always confused as to whether or not expiry is a duration or a timestamp because it's both: on the request object it's a duration, and when returned from this routine it's a timestamp. Perhaps changing this to `requestExpiresAt` would be a bit less confusing at all the callsites?

Reply:
Great suggestion! Thanks!